ui/selectors/selectors.js

```diff
@@ -826,6 +826,7 @@ function getAllowedAnnouncementIds(state) {
     10: Boolean(process.env.TOKEN_DETECTION_V2) && !process.env.IN_TEST,
     11: Boolean(process.env.TOKEN_DETECTION_V2) && !process.env.IN_TEST,
     12: true,
+    13: Boolean(process.env.ADD_POPULAR_NETWORKS),
```

Comment:
I guess with the feature flag addition we need to change this line
```suggestion
    13: true,
```